api/python/provisioner/commands/configure_setup.py

```diff
@@ -137,7 +137,8 @@ class StorageEnclosureParamsValidation:
 
     def __attrs_post_init__(self):
         params = attr.asdict(self)
-        if params['type'] == 'JBOD':
+        types = ['JBOD', 'virtual']
```

Comment:
Here we need:
['JBOD', 'RBOD', 'virtual', 'other']

Reply:
comment is addressed